src/Extension.ml

```diff
@@ -29,12 +31,19 @@ end
 module Instance = struct
   type t =
     { mutable client : LanguageClient.t option
+    ; statusBarItem : StatusBarItem.t
     ; duneFormatter : DuneFormatter.t
     }
 
-  let create () = { client = None; duneFormatter = DuneFormatter.create () }
+  let create () =
+    let statusBarItem =
+      Window.createStatusBarItem ~alignment:StatusBarAlignment.(tToJs Left) ()
+    in
+    statusBarItem##command#=selectSandboxCommandId;
+    { client = None; statusBarItem; duneFormatter = DuneFormatter.create () }
 
   let stop t =
+    StatusBarItem.hide t.statusBarItem;
```

Comment:
Why is this hide and not dispose?

Reply:
I thought It would be more efficient to reuse the StatusBarItem object. Now that I think about it, it does make more sense for `Instance.stop` to completely dispose its owned resources.

Just fixed it.